.github/workflows/ci.yml

```diff
@@ -11,22 +11,37 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  build_and_package:
+  windows:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu, windows, macos]
         raytracing_label: [raytracing, no-raytracing]
-        build_type: [standard]
-        include:
-          - egl_label: no-egl
-          - build_type: headless
-            os: ubuntu
-            raytracing_label: raytracing
-            egl_label: egl
 
-    runs-on: ${{matrix.os}}-latest
-    container: ${{ matrix.os == 'ubuntu' && 'ghcr.io/f3d-app/f3d-superbuild-ci' || null }}
+    runs-on: windows-latest
+
+    steps:
+
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        path: 'source'
+        submodules: true
+        lfs: true
+
+    - name: Build and package F3D
+      uses: ./source/.github/actions/f3d-superbuild/
+      with:
+        f3d_version: origin/master
+        raytracing_label: ${{matrix.raytracing_label}}
+  linux:
+    strategy:
+      fail-fast: false
+      matrix:
+        raytracing_label: [raytracing, no-raytracing]
+        egl_label: [no-egl, egl]
```

Comment:
do we want a no-raytracing egl build ?

I'm not against it, just checking this is intended.

Reply:
Any reason we wouldn't want it?

Reply:
As I said, not against it, just checking. It makes sense to ship it.